tests/sentry/sentry_metrics/limiters/test_cardinality_limiter.py

```diff
@@ -113,3 +120,57 @@ def test_accept_all(set_sentry_option):
     )
 
     assert not result.keys_to_remove
+
+
+def test_sample_rate_zero(set_sentry_option):
+    """
+    Assert that with a rollout rate of zero, no quotas are applied.
+    """
+
+    set_sentry_option(
+        "sentry-metrics.cardinality-limiter.limits.releasehealth.per-org",
+        [{"window_seconds": 3600, "granularity_seconds": 60, "limit": 0}],
+    )
+
+    set_sentry_option("sentry-metrics.cardinality-limiter.orgs-rollout-rate", 0.0)
+
+    backend = MockCardinalityLimiter()
+    backend.grant_hashes = 0
+    limiter = TimeseriesCardinalityLimiter("", backend)
+
+    result = limiter.check_cardinality_limits(
+        UseCaseKey.RELEASE_HEALTH,
+        {
+            PartitionIdxOffset(0, 0): {"org_id": 1, "name": "foo", "tags": {}},
+            PartitionIdxOffset(0, 1): {"org_id": 1, "name": "bar", "tags": {}},
+            PartitionIdxOffset(0, 2): {"org_id": 1, "name": "baz", "tags": {}},
+        },
+    )
+
+    assert not result.keys_to_remove
```

Comment:
Please assert you do not get any grants back in the result. That is what makes the difference between passing the rate limiter and having the rate limiter disabled.

Reply:
done

i previously configured the mock cardinality limiter to reject all timeseries, so i was thinking this assertion is good enough. but now it's more explicity